### DISCUSSION TITLE
Recommended system requirements for Docker + external database?

### DISCUSSION THREAD

**Discussion by empz** [tolgee/tolgee-platform#2477] [Q&A] [answered]
I'm looking into deploying Tolgee to Fly.io by leveraging the Docker image and using a external database.

I was wondering about the system requirements. Are the requirements expressed in the [docs](https://tolgee.io/platform/self_hosting/getting_started#system-requirements) for just the Java application or is it assumming an embedded database.

I'll be using this myself, one user and just one project. Do you think I could get a way with just 2 GB of memory for the application iself or is 4 GB really a hard requirement?

Thanks

**Comment by JanCizmar** [comment, marked as answer]
Hello! Thanks for the query. You can try it with 2GB, it might be fine.
